lstchain/io/dataselection.py

```diff
@@ -0,0 +1,227 @@
+from ctapipe.core import Component
+from ctapipe.core.traits import Int, Float, List
+from lstchain.reco.utils import filter_events
+
+import numpy as np
+import astropy.units as u
+from pyirf.binning import create_bins_per_decade  # , add_overflow_bins
+
+__all__ = ["DataSelection", "DataBinning"]
+
+
+class DataSelection(Component):
+    """
+    Collect various selection cuts to be applied for IRF production and
+    DL3 data reduction
+
+    Parameters for event filters will be combined in a dict so that the
+    filter_events() can be used.
+    """
+
+    intensity = List(
```

Comment:
I don't think it's a good idea to have traitlets for each feature here. It is very inflexible.

I think we should use a dict of cuts like @Bultako did here #573